libraries/AP_Compass/CompassCalibrator.h

```diff
@@ -45,8 +32,20 @@ class CompassCalibrator {
     // running is true if actively calculating offsets, diagonals or offdiagonals
     bool running() const;
 
+    // compass calibration states
+    enum class Status {
+        NOT_STARTED = 0,
```

Comment:
Pre exsisting, but this should really just use the MAVLink enum values as what its assigned to rather then hand writing them in.

Reply:
Changed to use the enumeration.

Reply:
.... and then reverted, as I decided pulling in the mavlink headers to this file wasn't worth it.